_15thnight/models.py

```diff
@@ -194,7 +198,7 @@ def get_alerts(cls):
     def get_provider_alerts(cls, provider):
         alerts = Alert.query \
             .join(cls.providers_notified) \
-            .filter(ProviderNotified.provider_id==provider.id) \
+            .filter(ProviderNotified.provider_id == provider.id) \
```

Comment:
I thought our coding style for this project was to follow standard pep8. Removal of those white spaces causes pep8 to through an E225.

http://pep8.readthedocs.io/en/latest/intro.html#error-codes